src/pages/Main/Main.scss

```diff
@@ -90,14 +86,12 @@
         #a2b6df 69.1%,
         #a2b6df 100%
       );
+      box-shadow: 0 3px 20px 0 rgba(0, 0, 0, 0.2);
       overflow: hidden;
 
-      @include tablet {
-        padding: 10 * 8px 0 0;
-      }
-
       @include desktop() {
         overflow: visible;
+        min-height: 487px;
```

Comment:
Can we use a multiple of `8px` here? `480` maybe?

Reply:
yup